packages/sdk/src/api/entities/VCLCryptoServicesDescriptor.ts

```diff
@@ -0,0 +1,8 @@
+import VCLInjectedCryptoServicesDescriptor from "./VCLInjectedCryptoServicesDescriptor";
+import VCLRemoteCryptoServicesUrlsDescriptor from "./VCLRemoteCryptoServicesUrlsDescriptor";
+
+export default class VCLCryptoServicesDescriptor {
+    constructor(
+        readonly injectedCryptoServicesDescriptor: Nullish<VCLInjectedCryptoServicesDescriptor> = null
```

Comment:
Probably it's better to remove the class `VCLInjectedCryptoServicesDescriptor` at all and put crypto services directly into VCLCryptoServicesDescriptor, because we don't have any other option. I'll edit the [ticket](https://velocitycareerlabs.atlassian.net/browse/SNJ-36).